README.md

```diff
@@ -75,6 +75,15 @@ parseURL('foo.com/foo?test=123#token', 'https://')
 // { protocol: 'https:', auth: '', host: 'foo.com', pathname: '/foo', search: '?test=123', hash: '#token' }
 ```
 
+### `stringifyParsedURL`
+```ts
```

Comment:
It would be nice to add a line break so that it can be consistent with other sections.

```suggestion
### `stringifyParsedURL`

```ts
```